dockerize.py

```diff
@@ -0,0 +1,20 @@
+# Prerequisites: Start a local Docker image with Runhouse installed using the following commands:
```

Comment:
Is this meant to be run as a test once the docker container is up? Maybe we should move this out of the main directory and into the tests file?

Reply:
Sounds good, done